Views/Players/PlayerInfo.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using dotnetNewProject.Models
+@model NewPlayer
+
+<ul>
+    @foreach (var player in ViewData["players"] as IList<NewPlayer>)

```

Comment:
Let's figure out how to pass this as a model instead of ViewData

Reply:
Opened new ticket for this.